lua/kubectl/resourcebuilder.lua

```diff
@@ -196,13 +196,24 @@ function ResourceBuilder:sort()
         end
         if type(valueA) == "table" and type(valueB) == "table" then
           if valueA.timestamp and valueB.timestamp then
-            return comp(tostring(valueA.timestamp), tostring(valueB.timestamp))
```

Comment:
I think we can do something prettier here, if you look at the utils.time.lua file, we do something like this with a timestamp:
```
  local parsed_time = vim.fn.strptime("%Y-%m-%dT%H:%M:%SZ", timestamp)

```

Reply:
done